diff --git a/source/3rd_party/CMakeLists.txt b/source/3rd_party/CMakeLists.txt
index b5656e1..f89b6d1 100644
--- a/source/3rd_party/CMakeLists.txt
+++ b/source/3rd_party/CMakeLists.txt
@@ -114,16 +114,16 @@ if(FFTW_FOUND)
   target_include_directories(cimg INTERFACE ${FFTW_INCLUDE_DIRS})
 endif()

-if (CMAKE_MODULE_PATH)
-  find_package(OpenCV QUIET)
-  if (OpenCV_FOUND)
-    target_compile_definitions(cimg INTERFACE cimg_use_opencv)
-    target_link_libraries(cimg INTERFACE ${OpenCV_LIBRARIES})
-    target_include_directories(cimg INTERFACE ${OpenCV_INCLUDE_DIRS})
-  endif()
-else()
-  message("No CMAKE_MODULE_PATH path for OpenCV configured")
-endif()
+# if (CMAKE_MODULE_PATH)
+#   find_package(OpenCV QUIET)
+#   if (OpenCV_FOUND)
+#     target_compile_definitions(cimg INTERFACE cimg_use_opencv)
+#     target_link_libraries(cimg INTERFACE ${OpenCV_LIBRARIES})
+#     target_include_directories(cimg INTERFACE ${OpenCV_INCLUDE_DIRS})
+#   endif()
+# else()
+#   message("No CMAKE_MODULE_PATH path for OpenCV configured")
+# endif()


 if(LIBAVCODEC_FOUND AND LIBAVFORMAT_FOUND AND LIBSWSCALE_FOUND AND LIBAVUTIL_FOUND)
